packages/expo-asset/src/AssetHooks.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+import { Asset } from './Asset';
+
+/**
+ * Load one or many of asset modules to use in your app.
+ * After (pre)loading these assets, you can use the module to load the downloaded asset in your app.
```

Comment:
I'm confused again.

> (pre)**loading**

and

> use the module to **load** the **downloaded**

somehow conflicts for me 🤔

Reply:
Yeah, I removed that from the docs, I didn't commit this change I think. While loading assets like these are often and widely used to "preload" bigger assets, it shouldn't be in here. Thanks for mentioning this.

Reply:
I just updated the description, does that make more sense?